app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
+        super().__init__(nickname)
+        self._favourite_dish = favourite_dish
+
+    def eat_favourite_dish(self) -> None:
+        print(f"{self.nickname} is eating {self._favourite_dish}")
```

Comment:
It should inherit from the ABC also, the same about Elf

![image](https://user-images.githubusercontent.com/104218550/218797291-150d0f39-a2e1-48f6-a591-bd276f0726e3.png)

Reply:
the interpreter prompted me to do this, but I don’t understand why we should do this if we don’t create abstract methods in this class

Reply:
Done